### ISSUE TITLE
NPM package lacks polyfill.js.map

### ISSUE THREAD

**Issue by IrregularShed** [developit/linkstate#2]
When using the polyfill (having installed from NPM) webpack throws this warning:

```
WARNING in ./~/linkstate/polyfill.js
Cannot find SourceMap 'polyfill.js.map': Error: Cannot resolve 'file' or 'directory' ./polyfill.js.map in C:\dev\project\node_modules\linkstate
```

For the time being I've just removed the reference to the source map from `polyfill.js` to stop the warnings scrolling past. I'm not sure how to fix it properly (otherwise I'd have a PR for you!)

**Comment by developit** [comment]
Ah, interesting. I must have deleted the map but not regenerated polyfill.js with sourcemaps turned off.  I think I'll just re-add the map, it's nice to have.

**Comment by developit** [comment]
Fixed in 1.0.1! https://unpkg.com/linkstate@1.0.1/

**Comment by IrregularShed** [comment]
You sir remain a legend 🎉
